qemu/tests/migration_multi_host.py

```diff
@@ -31,7 +31,7 @@ def run(test, params, env):
     class TestMultihostMigration(mig_type):
 
         def __init__(self, test, params, env):
-            super(TestMultihostMigration, self).__init__(test, params, env)
+            super(TestMultihostMigration, self).__init__(test, params, env, False)
```

Comment:
Because we want to continue use the vm by boot up from last test, if not set this value and use default value, will shutdown the vm and start the new vm.